doc/GUIDE.md

```diff
@@ -988,6 +988,14 @@ A complete list of all the supported command line options is given below:
 | `--include-[function,constant,struct,union,typedef,var]<String>` | Include a symbol of the given name and kind in the generated bindings. When one of these options is specified, any symbol that is not matched by any specified filters is omitted from the generated bindings. |
 | `--version`                                                  | print version information and exit |
 
+**macOS platform options (available only when running on macOS):**
+
+| Option                       | Meaning                                                                                                                                                                                                                                 |
```

Comment:
I'd say let's just add these two options to the list above and maybe put some comment (e.g. MacOS only). If we end up with many platform-specific options we can reconsider of course, but I think there's value in having all the options in the same place.